frontend/src/pages/about.tsx

```diff
@@ -18,7 +18,7 @@ export function AboutPage() {
           transcribee was founded from March until August 2023
           <img
             className="w-full"
-            src={pf_funding_svg}
+            // src={pf_funding_svg}
```

Comment:
Why that?

Reply:
Just WIP :) This needs to be moved/copied since we cannot import stuff from outside the frontend folder that easily when building the nix derivation

Reply:
fixed :)